service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/generator/C23EPODocumentParameterGeneratorTest.java

```diff
@@ -0,0 +1,162 @@
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Child;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.event.ManageOrdersEventData;
+import uk.gov.hmcts.reform.fpl.model.order.Order;
+import uk.gov.hmcts.reform.fpl.service.ChildrenService;
+import uk.gov.hmcts.reform.fpl.service.orders.docmosis.C23EPODocmosisParameters;
+import uk.gov.hmcts.reform.fpl.service.orders.docmosis.DocmosisParameters;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.enums.DocmosisTemplates.EPO;
+import static uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType.EMERGENCY_PROTECTION_ORDER;
+import static uk.gov.hmcts.reform.fpl.model.order.Order.C23_EMERGENCY_PROTECTION_ORDER;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.DATE_TIME_AT;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateTimeBaseUsingFormat;
+import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocumentReference;
+
+class C23EPODocumentParameterGeneratorTest {
+    private static final String LA_CODE = "LA_CODE";
+    private static final String LA_NAME = "Local Authority Name";
+    private static final String FURTHER_DIRECTIONS = "further directions";
+    private static final Order ORDER_TYPE = C23_EMERGENCY_PROTECTION_ORDER;
+
+    private static final Child CHILD = mock(Child.class);
+    public static final String CHILDREN_DESCRIPTION = "Children description";
+    public static final LocalDateTime APPROVAL_DATE_TIME = LocalDateTime.now().minusDays(8);
+    public static final LocalDateTime END_DATE_TIME = LocalDateTime.now();
+    public static final LocalDate EXCLUSION_DATE = LocalDate.now().plusDays(2);
+    public static final String EXCLUDE_PERSON = "Test user1";
+    public static final Address REMOVAL_ADDRESS = Address.builder()
+        .addressLine1("12 street").postcode("SW1").country("UK").build();
+
+    private final ChildrenService childrenService = mock(ChildrenService.class);
+    private final LocalAuthorityNameLookupConfiguration laNameLookup = mock(
+        LocalAuthorityNameLookupConfiguration.class
+    );
+
+    private final C23EPODocumentParameterGenerator underTest = new C23EPODocumentParameterGenerator(
```

Comment:
If you don't like the long lines here what you can do is annotate the class with `@ExtendWith({MockitoExtension.class})` and then annotate the mocks with `@Mock` and then annotate this field with `@InjectMocks`. With that you can remove all the initialisations if you want